modules/alphagov/manifests/init.pp

```diff
@@ -0,0 +1,18 @@
+# Installs all the Rubys
+#
+# The default list can be easily overridden in hiera/users/username.yaml
+# with something like:
+#       alphagov::repos:
+#           - spotlight
+#           - backdrop
+#
+class alphagov (
+  $repos   = [],
+) {
+
+  validate_array($repos)
+
+  if !(empty($repos)) {
```

Comment:
I've copied what `gds_ruby` does. If that's a bad example to copy, happy to change it.